Tests/test_PDB-MMCIFParser.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2017 by Francesco Gastaldello. All rights reserved.
+# Revisions copyright 2017 by Peter Cock.  All rights reserved.
+#
+# Converted by Francesco Gastaldello from an older unit test copyright 2002
```

Comment:
The new test looks nothing like the original self-test, so I don't think you need this note?